integration

```diff
@@ -72,7 +72,6 @@
   "Archef2000/homeassistant-upsplus",
   "ardevd/ha-bobcatminer",
   "arifwn/homeassistant-whatspie-integration",
-  "arjenvrh/audi_connect_ha",
```

Comment:
Do not remove repositories.
If that is needed, fill out this form <https://github.com/hacs/integration/issues/new?assignees=ludeeus&labels=flag&projects=&template=removal.yml>

Reply:
Does this apply when it is the same repo?  Just correcting a URL?  The one removed currently auto redirects to the one I'm trying to add. 

If so all good but want to confirm.